### PR TITLE
Support for Jumper Bumblebee

### PR DESCRIPTION
**Pull request by 3djc**
image

This PR brings support for Jumper Bumblebee (firmware and Companion) to V2.10 branch (given my current situation it may take some time for the support to the main branch to be submitted, but I'll try my best)

This radio features
2 x sliders
2 x 3Pos
2 x 2Pos
4 x toggle
6 customizable switches 

Currently only comes in 1W ELRS version, powered by two 18650 batteries.

### CODE REVIEW

File: radio/src/simu.cpp [hunk 1]
```diff
@@ -445,7 +445,7 @@ void OpenTxSim::updateKeysAndSwitches(bool start)
   SWITCH_KEY(C, 2, 3);
   SWITCH_KEY(D, 3, 3);
 
-  #if defined(RADIO_TPRO) || defined(RADIO_TPROV2) || defined(RADIO_TPROS)
+  #if defined(RADIO_TPRO) || defined(RADIO_TPROV2) || defined(RADIO_TPROS) || defined(RADIO_BUMBLEBEE)
```
Comment (pfeerick) [inline radio/src/simu.cpp:448]: This is for function switches, isn't it? i.e. TPRO S shouldn't be here?

**Comment by pfeerick** [comment]
Looks like there are two free/unused analogs on debug screen? i.e. A7 and A8

File: radio/src/simu.cpp (hunk 1 above)
Comment (3djc) [reply radio/src/simu.cpp:448]: > This is for function/customisable switches, isn't it? i.e. TPRO S shouldn't be here?

I'm afraid it is not; it defines the 'regular) switch types for the simu

**Comment by 3djc** [comment]
> Looks like there are two free/unused analogs on debug screen? i.e. A7 and A8

7 is VBAT, 8 is RTC Bat, you cannot move them, but they exist as ana in nonetheless

File: radio/src/simu.cpp (hunk 1 above)
Comment (pfeerick) [reply radio/src/simu.cpp:448]: Not quite. It actually does trigger some of the customisable switches, but only if those switches haven't been reached. i.e. For T-Pro 3 to 6 toggle the first 4 customisable switches. For TPro V2, 5 and 6 do the first two, etc. Just seems odd to switch to using 1-6 as the triggers for the (sometimes optional) E-F-G-H keys of those radios. But not relevant for this PR.

**Comment by pfeerick** [comment]
Hm... didn't notice that change... looks like it was a bug introduced in that PR that shall not be named... as they never used to be shown on the ana debug screen (i.e. they are **not** present on 2.9).  Which is even more funny considering the page does specifically say "Axis/Pots/Sliders" at the top... and last I checked RTC and Vbatt are most certainly not those :rofl: 

I think colorlcd only shows ADC_INPUT_MAIN + ADC_INPUT_FLEX, so displays only the controls it is supposed to. Something that can probably be fixed just in main for B&W at some point, as it purely cosmetic issue.

**Comment by 3djc** [comment]
Not too sure we should remove, maybe replace Axis/Pots/Slider by Analogs. If we remove them, next things that comes will require those to be there to help diagnose :D

**Comment by pfeerick** [comment]
Most likely. But, one way or the other, all the UIs should consistent. Another way to do it could be to move any non-input/calibrated analogs to their own page... which may actually be needed on colour as IIRC it maxes out at 10 analogs atm.

**Pull request merged**